browser/brave_profile_prefs.cc

```diff
@@ -33,7 +33,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       registry);
 
   // appearance
+#if !defined(OS_ANDROID)
   BraveThemeService::RegisterProfilePrefs(registry);
```

Comment:
There is a dark theme on android from Chromium I believe behind a feature flag, but it might not be using this. 
Doesn't block this PR in any case.